Libraries/StyleSheet/processAspectRatio.js

```diff
@@ -12,10 +12,13 @@
 
 const invariant = require('invariant');
 
-function processAspectRatio(aspectRatio: number | string): ?number {
+function processAspectRatio(aspectRatio?: number | string): ?number {
   if (typeof aspectRatio === 'number') {
     return aspectRatio;
   }
+  if (typeof aspectRatio !== 'string') {
```

Comment:
What if we only filter out `undefined`, `null` and `false`?

```suggestion
  if (typeof aspectRatio !== 'string' && !aspectRatio) {
```

Reply:
Sure I accept it and will more tests soon